schema/rusk.proto

```diff
@@ -25,16 +20,14 @@ message ExecuteStateTransitionRequest {
     repeated Transaction txs = 1;
     fixed64 height = 2;
     bytes seed = 3;
+    bool finalized = 4;
```

Comment:
Do we need `finalized` field for verifying a state transition request?

Reply:
We will need a `finalized` passed through to the VM in some form. In the state transition request it's ideal. See this [issue](https://github.com/dusk-network/rusk-vm/issues/263).

Reply:
Its a possibility, but maybe we don't want to mix these. `VerifyStateTransition` should probably take whatever latest state we have in Rusk - but then we need to discuss that in details. I think that would be a next step tho.

Reply:
If we include `finalized` then certainly we need to add the `state_root` anchor as well